bd-logger/src/async_log_buffer.rs

```diff
@@ -661,6 +684,15 @@ impl<R: LogReplay + Send + 'static> AsyncLogBuffer<R> {
     // stored in pre-config log buffer are guaranteed to be replayed before
     // the async log buffer goes back to processing incoming logs.
 
+    let mut current_feature_flags = self
+      .feature_flags_builder
+      .current_feature_flags()
+      .inspect_err(|e| {
+        // This should never fail unless there's a serious filesystem issue.
+        log::warn!("failed to load or create current feature flags: {e}");
+      })
+      .ok();
```

Comment:
Should we have this be lazily constructed in response to the first feature flag call we get? That way this is not initialized unless they make use of feature flags

Since this does file I/O we may want to wrap it in a `tokio::task::spawn_blocking` so that the I/O doesn't interfere with the event loop